src/schema-routes/schema-routes.ts

```diff
@@ -296,7 +296,10 @@ export class SchemaRoutes {
       return CONTENT_KIND.FORM_DATA;
     }
 
-    if (contentTypes.some((contentType) => contentType.includes("image/"))) {
+    if (
+      contentTypes.some((contentType) => contentType.includes("image/")) || 
+      contentTypes.some((contentType) => contentType.includes("application/octet-stream"))
+    ) {
       return CONTENT_KIND.IMAGE;
```

Comment:
We probably should rename this to `BLOB` instead of `IMAGE`, like [this](https://github.com/acacode/swagger-typescript-api/compare/main...TelenorCapture:swagger-typescript-api:brian/feat/support-blob?expand=1)